src/Dfe.ContentSupport.Web/Controllers/SitemapController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Dfe.ContentSupport.Web.Extensions;
+using Dfe.ContentSupport.Web.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Dfe.ContentSupport.Web.Controllers;
+
+[Route("/sitemap")]
```

Comment:
```suggestion
[Route("[controller]")]
```

Reply:
[1aad58a](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/811/commits/1aad58a750b7b64bb5a326a9a35751aafb3ffbc6)